RLTSquare/Ccq/Console/Command/CustomCommand.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RLTSquare\Ccq\Console\Command;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\MessageQueue\PublisherInterface;
+use RLTSquare\Ccq\Api\Data\QueueInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CustomCommand extends Command
+{
+    private const VARIABLE1 = '-var1';
+    private const VARIABLE2 = '-var2';
+
+    /**
+     * @var PublisherInterface
+     */
+    private PublisherInterface $publisher;
+    /**
+     * @var QueueInterface
+     */
+    protected QueueInterface $queue;
+    /**
+     * @var LoggerInterface
+     */
+    protected LoggerInterface $logger;
+
+    /**
+     * @param PublisherInterface $publisher
+     * @param QueueInterface $queue
+     * @param LoggerInterface $logger
+     */
+    public function __construct(
+        PublisherInterface $publisher,
+        QueueInterface $queue,
+        LoggerInterface     $logger
+    ) {
+        $this->publisher = $publisher;
+        $this->queue = $queue;
+        $this->logger = $logger;
+        parent::__construct();
+    }
+
+    /**
+     * @return void
+     */
+    protected function configure(): void
+    {
+        $this->setName('rltsquare:hello:world')->setDescription('Custom Command');
+        $this->setDefinition([
+            new InputArgument(self::VARIABLE1, InputArgument::REQUIRED, "Variable1"),
```

Comment:
Overlooked logic - The Symfony\Component\Console\Input\InputOption would be more suitable for task - see https://symfony.com/doc/current/console/input.html#using-command-arguments